src/lib/snarky/src/number.ml

```diff
@@ -38,9 +38,21 @@ module Make (Impl : Snark_intf.Basic) = struct
     assert (n < Field.size_in_bits) ;
     { upper_bound= Bignum_bigint.(pow2 n - one)
     ; lower_bound= Bignum_bigint.zero
-    ; var= Field.Checked.project bs
+    ; var= Field.Checked.pack bs
     ; bits= Some bs }
 
+  let mul_pow_2 n (`Two_to_the k) =
```

Comment:
I stared at this for a while until I realized that it was Lsb first. May want to at least put a comment above `to_bits`? Would be better to us `Bitstring.Lsb_first`

Reply:
made issue https://github.com/CodaProtocol/coda/issues/1153